app/views/rails_admin/main/edit.html.erb

```diff
@@ -1,3 +1,3 @@
-<%= rails_admin_form_for @object, url: edit_path(@abstract_model, @object.id), as: @abstract_model.param_key, html: { method: "put", multipart: true, class: "form-horizontal denser", data: { title: @page_name } } do |form| %>
+<%= rails_admin_form_for @object, url: edit_path(@abstract_model, @object.id), as: @abstract_model.param_key, html: { method: "put", multipart: true, class: "main", data: { title: @page_name } } do |form| %>
```

Comment:
`form-horizontal` is no more, and added `main` class to be used for applying our custom styles.